File: Core Java/studentset-app/src/com/techlabs/student/Student.java
package com.techlabs.student;

public class Student {

	private int rollNo;
	private int standard;
	private String name;
	
	public Student(int rollNo,int standard,String name)
	{
		this.rollNo=rollNo;
		this.standard=standard;
		this.name=name;
	}

	public int getRollNo() {
		return rollNo;
	}

	public int getStandard() {
		return standard;
	}

	public String getName() {
		return name;
	}

	@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + rollNo;
		result = prime * result + standard;
		return result;
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj)
			return true;
		if (obj == null)
			return false;
		if (getClass() != obj.getClass())
			return false;
		Student other = (Student) obj;
		if (rollNo != other.rollNo)
			return false;
		if (standard != other.standard)
			return false;
		return true;
	}
}

File: DOTNET/C#/StudentsetApp/StudentsetApp/Student.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace StudentsetApp
{
    class Student
    {
        private readonly int id;
        private readonly int standard;
        private readonly string studentName;

        public Student(int id, int standard, string studentName)
        {
            this.id = id;
            this.standard = standard;
            this.studentName = studentName;
        }

        public int Id {get { return id; } }
        public int Standard {get { return standard; } }
        public string StudentName {get { return studentName; } }

        public override bool Equals(object obj)
        {
            if(this == obj) { return true; }
            if(obj == null) { return false; }
            if(GetType() != obj.GetType()) { return false; }
            Student other = (Student) obj;
            if(Id != other.Id) { return false; }
            if(Standard != other.Standard) { return false; }
            return true;
        }

        public override int GetHashCode()
        {
            int prime = 31;
            int result = 1;
            result = prime * result + Id;
            result = prime * result + Standard;
            return result;

        }
    }
}
